api/v1alpha1/warehouse_types.go

```diff
@@ -233,9 +233,9 @@ type GitSubscription struct {
 	// For NewestFromBranch (commit filtering):
 	//   - `id`: The ID (sha) of the commit.
 	//   - `commitDate`: The commit date of the commit.
-	//   - `author`: The author of the commit message, in the format "Name <email>".
+	//   - `author`: The author of the commit message, in the format "Name email".
```

Comment:
I understand the angled brackets are probably causing some trouble for doc generation.

If the angled brackets here were indicative of some kind of placeholder, as they often tend to be, I think this change wouldn't be so bad, but unfortunately, that's not the case here and the angled brackets are quite literally expected to be included in the string the user provides as the value of this field.

Example from an actual git commit: `Author: Kent Rancourt <kent.rancourt@gmail.com>`.

So, unfortunately, I think we're significantly reducing the accuracy of the docs with this change. Is there any possibility of encoding/escaping in the templates to account for this instead?

Reply:
For doc generation no but for docusauras compilation yes. It interprets <email> as a tag and fails to compile due to what it thinks is the absence of a closing tag.

I fixed using `&lt;` + `&gt;` : https://github.com/akuity/kargo/pull/4901/commits/fbe346653b0f9e677d39ec1f343504264ead7a69